tests/src/test/java/edu/snu/onyx/tests/runtime/master/scheduler/FaultToleranceTest.java

```diff
@@ -196,18 +214,14 @@ public void testContainerRemoval() throws Exception {
       if (stage.getScheduleGroupIndex() == 0) {
 
         // There are 3 executors, each of capacity 2, and there are 6 TaskGroups in ScheduleGroup 0.
-        RuntimeTestUtil.mockSchedulerRunner(pendingTaskGroupQueue, schedulingPolicy, false);
+        RuntimeTestUtil.mockSchedulerRunner(pendingTaskGroupQueue, schedulingPolicy, jobStateManager, false);
         assertTrue(pendingTaskGroupQueue.isEmpty());
-        stage.getTaskGroupList().forEach(taskGroup -> {
-          jobStateManager.onTaskGroupStateChanged(taskGroup, TaskGroupState.State.EXECUTING);
+        stage.getTaskGroupList().forEach(taskGroup ->
           RuntimeTestUtil.sendTaskGroupStateEventToScheduler(scheduler, containerManager,
-              taskGroup.getTaskGroupId(), TaskGroupState.State.COMPLETE, 1, null);
-            });
+              taskGroup.getTaskGroupId(), TaskGroupState.State.COMPLETE, 1, null));
```

Comment:
Can you create another `sendTaskGroupStateEventToScheduler()` without the `TaskGroupState.RecoverableFailureCause cause)` argument such that this line can be replaced with the following line?

```java
taskGroup.getTaskGroupId(), TaskGroupState.State.COMPLETE, 1) // no null here
```